src/bin/wallet.rs

```diff
@@ -32,25 +32,10 @@ fn print_usage(program: &str, opts: Options) {
 
 fn main() -> io::Result<()> {
     env_logger::init();
-    if is(Stream::Stdin) {
-        eprintln!("nothing found on stdin, expected a json file");
-        exit(1);
-    }
-
-    let mut buffer = String::new();
-    let num_bytes = io::stdin().read_to_string(&mut buffer).unwrap();
-    if num_bytes == 0 {
-        eprintln!("empty file on stdin, expected a json file");
-        exit(1);
-    }
-
-    let id: Mint = serde_json::from_str(&buffer).unwrap_or_else(|e| {
-        eprintln!("failed to parse json: {}", e);
-        exit(1);
-    });
 
     let mut opts = Options::new();
     opts.optopt("l", "", "leader", "leader.json");
+    opts.optopt("m", "", "mint", "mint.json");
```

Comment:
Let's dump `mint.json` altogether. Instead add options for `--request-tokens=NUM` and `--pkcs8-file=PATH`. See `ring`'s documentation. After reading the file, you're run:

```rust
let key_pair =
   signature::Ed25519KeyPair::from_pkcs8(
            untrusted::Input::from(&pkcs8_bytes))?;
```